lib/membrane_element_rtp_aac/depayloader.ex

```diff
@@ -11,17 +11,11 @@ defmodule Membrane.RTP.AAC.Depayloader do
   def_input_pad :input, accepted_format: %RTP{}
   def_output_pad :output, accepted_format: %AAC{encapsulation: :none}
 
-  def_options profile: [
-                default: :LC
-              ],
-              sample_rate: [
-                default: 44_100
-              ],
-              channels: [
-                default: 2
-              ],
-              mode: [
-                spec: Utils.mode()
+  def_options mode: [
+                spec: Utils.mode(),
+                description: """
+                The bitrate mode that dictates the maximum length of a single frame. For more information refer to Utils.mode()'s typedoc.
```

Comment:
```suggestion
                The bitrate mode that dictates the maximum length of a single frame. For more information refer to `Utils.mode()`'s typedoc.
```
Putting the module name into a quote (`...`) adds a link in the documentation, so it's worth adding